src/components/HighlightedText/__test__/HighlightedText.spec.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import styled from 'styled-components';
+import HighlightedText from '../';
+
+describe('<HighlightedText />', () => {
+    const parts = [
+        {
+            value: 'Apples',
+            type: 'text',
+        },
+        {
+            value: 'varieties',
+            type: 'hit',
+        },
+        {
+            value: 'Honeycrisp',
+            type: 'text',
+        },
+    ];
+    it('should return 3 <span> with theirs respective values because the default wrapper is a span', () => {
+        const component = mount(<HighlightedText parts={parts} />);
+        const container = component.find('span');
+        expect(
+            container
+                .at(0)
+                .html()
+                .includes('Apples'),
+        ).toBe(true);
+        expect(
+            container
+                .at(1)
+                .html()
+                .includes('varieties'),
+        ).toBe(true);
+        expect(
+            container
+                .at(2)
+                .html()
+                .includes('Honeycrisp'),
+        ).toBe(true);
```

Comment:
I think it would be cleaner like this

```js
const component = mount(<HighlightedText parts={parts} />);
        const container = component.find('span');
        expect(container.length).toBe(3);
        parts.forEach(({ value }, index) => {
            expect(
                container
                    .at(index)
                    .html()
                    .includes(value),
            ).toBe(true);
        });
```